OpenRA.Mods.Common/Widgets/Logic/Editor/MapEditorLogic.cs

```diff
@@ -71,12 +71,15 @@ public MapEditorLogic(Widget widget, World world, WorldRenderer worldRenderer)
 				};
 			}
 
-			var copyOverlayDropdown = widget.Get<DropDownButtonWidget>("OVERLAY_BUTTON");
-			copyOverlayDropdown.OnMouseDown = _ =>
+			var overlayDropdown = widget.Get<DropDownButtonWidget>("OVERLAY_BUTTON");
```

Comment:
`widget.GetOrNull`, as `Get` will throw if it's not found.

Reply:
Fixed. I had copied the code from `copyFilterDropdown`, it appears it is the only non-optional button in left MapEditorLogic